releases/point-2023-2-release-note.md

```diff
@@ -0,0 +1,86 @@
+    * **Receive an email that opens the request in SysKit Point Teams app** 
+    * **Be able to resolve access and provisioning requests within SysKit Point Teams app**
+In the newest addition, you will:
+    * **Get a task in the SysKit Point app as well**
+    * **Be able to navigate to a task in the SysKit Point web app from the received email**
+    * **Be able to resolve access and provisioning requests within the SysKit Point web app**
+
+* **Group email address and site address editing added to provisioning requests**!
+    * **When requesting a Microsoft Team or a Microsoft 365 Group**, you can **edit the group email address**. 
+    * **When requesting a new site**, you can now **edit the site address**, thus affecting the site's URL.
+
+## Improvements & Bug Fixes
+
+* **SysKit Point sync was improved to retrieve additional properties to distinguish between cloud and on-premises groups synced to Azure**! 
+    * **Refreshed look with additional icons** and **new on-premises views** on reports help you differentiate between the two group types
+    * **Microsoft Teams & Groups inventory report** and **Group Membership** report enable filtering by group type with the help of the newly added views:
+        * **Distribution Lists** and **Distribution Lists (On-Premises)**
+        * **Security Groups** and **Security Groups (On-Premises)**
+        * **Mail-Enabled Security Groups**
+
+* **Performed by filter added to Audit Logs Overview report**! The new filter enables you to generate the report showing audit logs performed by:
+    * **all users**
+    * **only internal users**
+    * **only external users**
+    * **specific users**; here, you can select one or more users from your tenant in the people picker dialog
+
+* **Recycled folder activity support added**!
+    * The audit logs related to the Recycled folder activity are now collected in SysKit Point by default.
+    * Activity is visible in the File and Page Activities report.
+    * Activity triggers the Changed File or Folder alert if the alert is enabled.
+
+* **Reports improved**!
+    * **Job Title column was added** to the Users inventory report.
+    * **Last Logged In column was added** to the Users with Privileged Access report.
+
+* **Multiple sync improvements and fixes are available**.
+    * **Fixed issues where changes in group memberships were not synced**.
+    * **Added support for Request files links**. Before the changes, the following error would appear in the error logs: `SysKit.Point.WorkerUtils.Database.CustomSqlException: Violation of PRIMARY KEY constraint 'PK_Crawl_SharePointSharingLinks'. Cannot insert duplicate key in object 'Crawl.SharePointSharingLinks'.`
+    * **Fixed an issue** that would appear when syncing sites with more than 1000 lists containing an Acess Requests list. The following exception was thrown: `Violation of PRIMARY KEY constraint 'PK_Crawl_SharePointObjects'. Cannot insert duplicate key in object 'Crawl.SharePointObjects'.`
```

Comment:
access

Reply:
instead of Acess

Reply:
fixed. not sure why grammarly didn't throw anything here.